dockstore-client/src/main/java/io/dockstore/client/cli/nested/NotificationsClients/SlackMessage.java

```diff
@@ -0,0 +1,18 @@
+package io.dockstore.client.cli.nested.NotificationsClients;
+
+/**
+ * @author gluu
+ * @since 19/01/18
+ */
+public class SlackMessage {
+    String text;
+    String username = "Dockstore CLI";
+
```

Comment:
you'll want the UUID to be a field in here to be picked up into the JSON
also if there is a username field, make it pick up the actual username

Reply:
done